x/stake/client/cli/query.go

```diff
@@ -170,7 +170,7 @@ func GetCmdQueryValidatorRedelegations(queryRoute string, cdc *codec.Codec) *cob
 			}
 
 			res, err := cliCtx.QueryWithData(
-				fmt.Sprintf("custom/%s/validatorRedelegations", queryRoute),
+				fmt.Sprintf("custom/%s/validatorRedelegations", storeKey),
```

Comment:
the query route and the store key don't have to be the same, but that seems like good practice for now :+1:

Reply:
Agree.